packages/reactivity/src/effectScope.ts

```diff
@@ -119,17 +119,24 @@ export class EffectScope {
     if (this._active) {
       this._active = false
       let i, l
-      for (i = 0, l = this.effects.length; i < l; i++) {
-        this.effects[i].stop()
+      const effects = this.effects.slice()
```

Comment:
this change was for https://github.com/vuejs/core/issues/5783. it's not necessary since #5783 fixed via https://github.com/vuejs/core/commit/21932840eae72ffcd357a62ec596aaecc7ec224a